src/pages/home/report/ReportActionItem.tsx

```diff
@@ -933,7 +933,7 @@ function ReportActionItem({
         <PressableWithSecondaryInteraction
             ref={popoverAnchorRef}
             onPress={draftMessage === undefined ? onPress : undefined}
-            style={[action.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE ? styles.pointerEventsNone : styles.pointerEventsAuto]}
+            style={[action.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE && !isDeletedParentAction ? styles.pointerEventsNone : styles.pointerEventsAuto]}
```

Comment:
I see that we also have `isDeletedParentAction`
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/libs/ReportActionsUtils.ts#L115-L117

which is very similar to `isMessageDeleted`.
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/libs/ReportActionsUtils.ts#L804-L806

I decided to use `isDeletedParentAction` instead which is already defined at the top of the component.